packages/opentelemetry-instrumentation-azure-search/test_trace.py

```
@@ -0,0 +1,26 @@
+client._index_name = "demo-index"
+client._endpoint = "https://demo.search.windows.net"
+
+with patch("azure.core.pipeline.Pipeline.run", return_value=MagicMock(
+    http_response=MagicMock(status_code=200, text=lambda encoding=None: '{"value": []}', headers={})
+)):
+    try:
+        client.search("azure openai", top=5)
+    except Exception:
+        pass
+
+print("Done")
```

Ruff (0.15.17)

[warning] No newline at end of file

Add trailing newline

(W292)